bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -206,7 +200,7 @@
 {% block email_form %}{% endblock %}
 
 {% block js %}
-  {% if not l10n_has_tag('emerging_tech_links') %}
+  {% if LANG.startswith('en-') or not l10n_has_tag('emerging_tech_links') %}
```

Comment:
I think instead of the video button for the fallback, we can have the same Rust link as the CTA and use the existing "Learn more" string.